app/controllers/concerns/verify_profile_concern.rb

```diff
@@ -3,7 +3,13 @@ module VerifyProfileConcern
 
   def url_for_pending_profile_reason
```

Comment:
We only get to this if the user has a pending profile and singing in without SP.